lld/COFF/Driver.cpp

```diff
@@ -737,7 +737,7 @@ StringRef LinkerDriver::mangleMaybe(Symbol *s) {
   // If we find a similar mangled symbol, make this an alias to it and return
   // its name.
   log(unmangled->getName() + " aliased to " + mangled->getName());
-  unmangled->weakAlias = ctx.symtab.addUndefined(mangled->getName());
+  unmangled->setWeakAlias(ctx.symtab.addUndefined(mangled->getName()));
```

Comment:
As the `weakAlias` member hasn't been made private here, I presume, and we're passing the default of false to the `antiDep` parameter, I guess these changes strictly are unnecessary? It's of course nice for consistency to add them, but if working on the codebase going forward, there's no guarantee that new calls will use this form, as they may just as well just assign to `weakAlias`?

Reply:
We could make it private, but it will need a few more changes. As `getWeakAlias` follows the chain, it's not a replacement for accessing `weakAlias` directly. I will create a follow-up, thanks!

Reply:
I'm not sure if that's really necessary - I was just checking I understood the situation correctly :-)